arangod/Aql/Aggregator.cpp

```diff
@@ -933,6 +933,35 @@ struct AggregatorBitXOr : public AggregatorBitFunction<BitFunctionXOr> {
       : AggregatorBitFunction(opts) {}
 };
 
+struct AggregatorMergeLists : public Aggregator {
+  explicit AggregatorMergeLists(velocypack::Options const* opts)
+      : Aggregator(opts) {}
+
+  ~AggregatorMergeLists() { reset(); }
+
+  // cppcheck-suppress virtualCallInConstructor
```

Comment:
good question, this was copied from other aggregators and I wondered as well. This might be a historic artifact. Probably the constructor of the base class `Aggregator` did call `reset` in its constructor to ensure that everything is properly set up. But this of course was going to fail and was changed.